wagtailio/newsletter/templates/newsletter/newsletter_index_page.html

```diff
@@ -16,34 +16,41 @@ <h1 class="page-heading__heading">This week in Wagtail</h1>
         </header>
 
         <div class="container">
-            <!-- Begin Mailchimp Signup Form -->
-            <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
-            <style type="text/css">
-                #mc_embed_signup{clear:left; font:14px Helvetica,Arial,sans-serif; }
-                /* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
-                We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
-            </style>
-            <div id="mc_embed_signup">
-            <form action="https://torchbox.us1.list-manage.com/subscribe/post?u=fa038e191e9c77563d1c3bfa4&amp;id=ac422ce60a" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+        <!-- Begin Mailchimp Signup Form -->
```

Comment:
Could you indent this all one tab so it matches the original indentation? It makes the diff less noisy as a lot of html didn't  change.

Reply:
Will do.